audit/pkg/config/debug.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+// Debug defines the available debug configuration.
+type Debug struct {
+	Addr   string `ocisConfig:"addr" env:"NOTIFICATIONS_DEBUG_ADDR"`
```

Comment:
should be `env:"AUDIT_...."`, needs also be change in some more config structs

Reply:
copy and paste :) sorry, fixed now